Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/docrepository/adapter/AdapterComponentDocRepositoryOrchImpl.java

```diff
@@ -554,6 +559,16 @@ protected DocumentMetadata setDocument(
         return null;
     }
 
+    /**
+     * Retrieve from adapter.properties. If it doesn't exist, return default value
```

Comment:
Documentation suggests logic that doesnt exist. It will always return the default value.